packages/discord.js/typings/index.d.ts

```diff
@@ -4094,6 +4112,26 @@ export class EntitlementManager extends CachedManager<Snowflake, Entitlement, En
   public consume(entitlementId: Snowflake): Promise<void>;
 }
 
+export interface FetchSubscriptionOptions extends BaseFetchOptions {
+  cache?: boolean;
+  sku: SKUResolvable;
+  subscriptionId: Snowflake;
+}
```

Comment:
`BaseFetchOptions` already has `cache`:

https://github.com/discordjs/discord.js/blob/3669d5e1120ed0a6e8d8cb8c48414d7841efd594/packages/discord.js/typings/index.d.ts#L4964-L4967

```suggestion
export interface FetchSubscriptionOptions extends BaseFetchOptions {
  sku: SKUResolvable;
  subscriptionId: Snowflake;
}
```